kotlinx-coroutines-core/jvm/src/internal/MainDispatchers.kt

```diff
@@ -67,7 +67,7 @@ public fun MainCoroutineDispatcher.isMissing(): Boolean = this is MissingMainCor
 @Suppress("MayBeConstant")
 private val SUPPORT_MISSING = true
 
-@Suppress("ConstantConditionIf")
+@Suppress("ConstantConditionIf", "IMPLICIT_NOTHING_TYPE_ARGUMENT_AGAINST_NOT_NOTHING_EXPECTED_TYPE") // KT-47626
```

Comment:
```suggestion
@Suppress(
    "ConstantConditionIf",
    "IMPLICIT_NOTHING_TYPE_ARGUMENT_AGAINST_NOT_NOTHING_EXPECTED_TYPE" // KT-47626
)
```
This way, it's clearer what the comment refers to.